packages/sdk/src/sdk-blockchains/ethereum/common/index.ts

```diff
@@ -42,6 +53,68 @@ export function convertToEthereumAssetType(assetType: AssetType): EthAssetType |
 				contract: convertToEthereumAddress(assetType.contract),
 			}
 		}
+		case "ERC721": {
+			return {
+				assetClass: "ERC721",
+				contract: convertToEthereumAddress(assetType.contract),
+				tokenId: assetType.tokenId,
+			}
+		}
+		case "ERC721_Lazy": {
+			return {
+				assetClass: "ERC721_LAZY",
+				contract: convertToEthereumAddress(assetType.contract),
+				tokenId: assetType.tokenId,
+				uri: assetType.uri,
+				creators: assetType.creators.map(c => ({
+					account: convertToEthereumAddress(c.account),
+					value: toBn(c.value).toNumber(),
+				})),
+				royalties: assetType.royalties.map(r => ({
+					account: convertToEthereumAddress(r.account),
+					value: toBn(r.value).toNumber(),
+				})),
+				signatures: assetType.signatures.map(str => toBinary(str)),
+			}
+		}
+		case "ERC1155": {
+			return {
+				assetClass: "ERC1155",
+				contract: convertToEthereumAddress(assetType.contract),
+				tokenId: assetType.tokenId,
+			}
+		}
+		case "ERC1155_Lazy": {
+			return {
+				assetClass: "ERC1155_LAZY",
+				contract: convertToEthereumAddress(assetType.contract),
+				tokenId: assetType.tokenId,
+				uri: assetType.uri,
+				supply: assetType.supply !== undefined ? toBigNumber(assetType.supply): toBigNumber("1"),
+				creators: assetType.creators.map(c => ({
+					account: convertToEthereumAddress(c.account),
+					value: toBn(c.value).toNumber(),
+				})),
+				royalties: assetType.royalties.map(r => ({
```

Comment:
Maybe add new utility for mapping from one to another model
you copy-pasting same code everywhere